File: bench/fixtures/multilang_game/engine/c/arena_reward.c
#include "arena_reward.h"
#include <stdio.h>

/* ClaimArenaReward engine binding */
void claim_arena_reward(long player_id) {
    const char *key = "arena_reward";
    int id = ARENA_REWARD_ID; /* 100126 */
    (void)player_id; (void)key; (void)id;
}

File: bench/fixtures/multilang_game/client/ts/arenaReward.ts
// ClaimArenaReward / arena_reward / 100126
export function ClaimArenaReward(playerId: number): Promise<void> {
  const cfgId = 100126;
  const key = "arena_reward";
  // TS_ONLY_CLUE_SeasonPassArenaBadge
  return fetch("/api/claim", {
    method: "POST",
    body: JSON.stringify({ playerId, cfgId, key }),
  }).then(() => undefined);
}
